manifests/charts/istiod-remote/values.yaml

```diff
@@ -33,10 +33,6 @@ sidecarInjectorWebhook:
   objectSelector:
     enabled: false
     autoInject: true
-  
```

Comment:
Good that you removed them - please add instead 'injectionURL'. Using a remote URL for injection is not specific to 'centralIstiod' - and doesn't have to point to istiod, can be any URL.

Reply:
updated